Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 name = "bech32"
 version = "0.9.1"
-authors = ["Clark Moody", "The rust-bitcoin developers"]
+authors = ["Clark Moody", "Andrew Poelstra", "Tobin Harding", "The rust-bitcoin developers"]
 repository = "https://github.com/rust-bitcoin/rust-bech32"
 documentation = "https://docs.rs/bech32/"
-description = "Encodes and decodes the Bech32 format"
+description = "Encodes and decodes the Bech32 format (bech32 and bech32m)"
```

Comment:
In 310012dd3e5f5725b5bde9021606ed1d5542620d:

I'd say "and implement the bech32 and bech32m checksums" rather than putting these in parens.